### PR TITLE
Release/9152-Publish-failing-screenshots-in-general-channel

### PR DESCRIPTION
**Pull request by 30mar**
## Description
Publish failing screenshots in the general teams channel

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9152

## Screenshots
Screenshot 2021-09-29 at 01 52 16

### CODE REVIEW

File: scripts/announce_failing_screenshots.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  curl --output /dev/null --silent $Failed_SCREENSHOTS_TEAMS \
+  -H 'Content-Type: application/json' \
+  --data-binary @- << EOF
+{
+  "@type": "MessageCard",
+  "@context": "http://schema.org/extensions",
+  "themeColor": "0076D7",
+  "summary": "screenshots have failed!",
+  "sections": [{
+      "activityTitle": "screenshots have failed!",
```
Comment (naveeddotio) [inline scripts/announce_failing_screenshots.sh:19]:
```suggestion
      "activityTitle": "Screenshots have been failed!",
```

**Review by naveeddotio — APPROVED** [review]

File: scripts/announce_failing_screenshots.sh (hunk 1 above)
Comment (30mar) [reply scripts/announce_failing_screenshots.sh:19]: I think have failed is correct

Comment (marcussc) [reply scripts/announce_failing_screenshots.sh:19]: Hmm I am not an expert with English Grammar but this sounds strange to me. `"activityTitle": "Screenshots have failed!",` would be my idea.

Comment (naveeddotio) [reply scripts/announce_failing_screenshots.sh]: Edited my suggestion, totally agree - I realised the mistake on the second read.

**Review by CV113 — APPROVED** [review]
UA approved

**Review by marcussc — APPROVED** [review]

**Pull request merged**